projects/MuonG2.yaml

```
@@ -0,0 +1,10 @@
+Department: Physics
+Description: 'Monte Carlo simulations of beam dynamics for the Muon g-2 experiment'
+ID: '541'
+Name: MuonG2
+Organization: Fermilab
+PIName: Ian Bailey
+Sponsor:  Lauren Michael
```

```suggestion
Sponsor:
```
Sponsor is not a single-line field, but a block that includes the CampusGrid... stuff inside it.

I'm not sure why you want to put your name in there -- if you just want to record that you created the project, that information is already in the GitHub pull request.

Also, I notice that the organization is Fermilab, but it's a campus grid? That seems a little odd to me... usually Fermilab projects are under the Fermilab VO, i.e.
```
Sponsor:
  VirtualOrganization:
    ID: '9'
    Name: Fermilab
```

Also also, I notice that Muon g-2 already has a project entry at https://github.com/opensciencegrid/topology/blob/master/projects/gm2.yaml.